docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  mysql:
+    image: mysql:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: mysql-container
+    restart: always
+    ports:
+      - "3306:3306"
+    networks:
+      - db-data-net
+    volumes:
+      - db-data:/var/lib/mysql
+  pythonapp:
```

Comment:
Add dependency on the database that includes database health check, because booting up database may take more time and your backend server will fail to connect to it.